plugin/trino-iceberg/src/main/java/io/trino/plugin/iceberg/IcebergTableProperties.java

```diff
@@ -114,4 +135,21 @@ private static void validateFormatVersion(int version)
                     format("format_version must be between %d and %d", FORMAT_VERSION_SUPPORT_MIN, FORMAT_VERSION_SUPPORT_MAX));
         }
     }
+
+    @SuppressWarnings("unchecked")
+    public static List<String> getOrcBloomFilterColumns(Map<String, Object> tableProperties)
+    {
+        List<String> orcBloomFilterColumns = (List<String>) tableProperties.get(ORC_BLOOM_FILTER_COLUMNS);
+        return orcBloomFilterColumns == null ? ImmutableList.of() : ImmutableList.copyOf(orcBloomFilterColumns);
+    }
+
+    @SuppressWarnings("unchecked")
+    public static Double getOrcBloomFilterFpp(Map<String, Object> tableProperties)
+    {
+        Double fpp = (Double) tableProperties.get(ORC_BLOOM_FILTER_FPP);
+        if (fpp < 0.0 || fpp > 1.0) {
```

Comment:
Same as the Hive version, use the `validation` argument on `doubleProperty`

Reply:
This is written to keep consistent with hive connector bloom filter. Are you sure you want to modify it?

Reply:
This place is used to receive and verify the FPP value specified in create table. At present, the consumer < double > validation method is not used for the verification related to create table.Maybe @Praveen2112 can give more information.

Reply:
> Same as the Hive version, use the `validation` argument on `doubleProperty`

Done.

Reply:
That could be nice, putting these into a class like `OrcBloomFilterTablePeroperties`
```
    public static final String ORC_BLOOM_FILTER_COLUMNS = "orc_bloom_filter_columns";
    public static final String ORC_BLOOM_FILTER_FPP = "orc_bloom_filter_fpp";

    public static final PropertyMetadata BLOOM_FILTER_FPP_PROPERTY = doubleProperty(
                        ORC_BLOOM_FILTER_FPP,
                        "ORC Bloom filter false positive probability",
                        orcWriterConfig.getDefaultBloomFilterFpp(),
                        IcebergTableProperties::validateOrcBloomFilterFpp,
                        false)
    public static final PropertyMetadata BLOOM_FILTER_COLUMNS_PROPERTY = ...
...